pkg/util/use_bare_configs.go

```diff
@@ -0,0 +1,31 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package util
+
+import "github.com/DataDog/datadog-agent/pkg/config"
+
+// set in testing to force the result of CcaInAD
+var forcedCcaInAD *bool
+
+// ForceCcaInAD forces a choice for CcaInAD, and returns
+// a function to revert the change, suitable for use in a `defer`.
+func ForceCcaInAD(use bool) func() {
```

Comment:
This does not appear to be used anywhere?

Reply:
It will be (for tests).  For this particular commit it's not required, as it only chooses a ConfigManager implementation.

Reply:
fair enough -- I'll drop it.